docs/production-deployment/cloud/api-keys.mdx

```diff
@@ -197,55 +200,102 @@ tcld apikey disable --id <api-key-id>
 tcld apikey enable --id <api-key-id>
 ```
 
-## API Keys for Client Connections
+## Managing API Keys for Service Accounts {#serviceaccount-api-keys}
 
-Temporal Cloud supports secure programmatic access through API Key Authentication.
+Global Administrators have the ability to manage API Keys for any Service Account in their Temporal Cloud Account.
+The process to manage API Keys for Service Accounts is very similar to the process of managing API Keys for Users with one difference: Unlike User API Keys, Global Admins are able to generate API Keys for Service Accounts.
+Disabling/enabling, deleting, and viewing API Keys for Service Accounts follows the same process as API Keys for Users.
+
+### Create an API Key for a Service Account
+
+Create an API Key for a Service Account using the Temporal Cloud UI or tcld.
+
+#### Using the Cloud UI
 
-### Overview
+1. Navigate to https://cloud.temporal.io/settings/api-keys
+2. Click the Create API Key button
+3. Click Service Account at the top of the page under "Create an API Key for*"
```

Comment:
```suggestion
3. Click Service Account at the top of the page under "Create an API Key for"
```

Reply:
Is there a * here or maybe a typo?

Reply:
If quotes are used here, they should be used consistently through this list and the document. When wording is taken from UIs, make sure they match exactly including punctuation such as colons and ellipses.